osds/verify.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2019 The OpenSDS Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if ! [ -f "./bin/gometalinter" ]; then echo "Download The files";
+	curl -L https://git.io/vp6lP | sh
+fi
+export PATH=${PATH}:${GOPATH}/bin:${GOPATH}/src/github.com/opensds/opensds/osds/bin
+gometalinter --disable-all --enable=gofmt --enable=misspell --enable=golint --exclude=vendor --exclude=test ./...
+if [ $? != 0 ]; then
+       	echo "Please fix the warnings!"
+	echo "Run ./update-gofmt.sh if any warnings in gofmt"
+       	exit 1
+else
+	echo "Gofmt,misspell,golint checks have been passed"
+fi	
```

Comment:
didn't see any golink, misspell check in CI log

Reply:
Fixed & Updated verify.sh